src/components/PopupWithForm.js

```diff
@@ -19,6 +20,10 @@ export default class PopupWithForm extends Popup {
     return this._formValues;
   }
 
+  handleAvatarChange(link) {
+    this.avatarPic.src = link;
+  }
+
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // Here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`